Gopkg.toml

```diff
@@ -24,19 +24,43 @@ required = [
   name="github.com/metal3-io/baremetal-operator"
   branch="master"
 
+[[override]]
+  name = "k8s.io/kube-aggregator"
+  version = "kubernetes-1.14.1"
+
+[[override]]
+  name = "k8s.io/code-generator"
+  version = "kubernetes-1.14.1"
+
+[[override]]
+  name = "k8s.io/api"
+  version = "kubernetes-1.14.1"
+
+[[override]]
+  name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.14.1"
+
+[[override]]
+  name = "k8s.io/apimachinery"
+  version = "kubernetes-1.14.1"
+
+[[override]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.14.1"
+
 # STANZAS BELOW ARE GENERATED AND MAY BE WRITTEN - DO NOT MODIFY BELOW THIS LINE.
 
 [[constraint]]
   name="sigs.k8s.io/controller-runtime"
-  version="v0.1.1"
+  branch="release-0.2"
```

Comment:
Do you think it matters if we use the release-0.2 branch or point to the specific sha everyone else seems to be using? e.g. https://github.com/openshift/cluster-api-provider-aws/blob/master/Gopkg.toml#L56, https://github.com/metal3-io/baremetal-operator/blob/master/Gopkg.toml#L48

Reply:
I followed what they did here, since it is a direct dep for us: https://github.com/openshift/cluster-api/blob/openshift-4.2-cluster-api-0.1.0/Gopkg.toml#L48